docs/guide/migrating-to-version-9.md

```diff
@@ -224,6 +251,41 @@ Component has been renamed to `KDropdown`
 
 ### KMultiselect
 
+#### Props
+
+* `testMode` prop has been removed
+* `filterFunc` prop name has been changed to `filterFunction`
+* `disabledTooltipText` property of object passed through `items` prop is no longer supported
+* default value of `collapsedContext` prop has changed to `true`
+* default value of `selectedRowCount` prop has changed to `1`
+
+#### Events
+
+* `item:added` event was renamed to `item-added`
+* `item:removed` event was renamed to `item-removed`
```

Comment:
why?

Reply:
We don't have this pattern to use colons in event names anywhere in already reskinned components - I thought using `-` is pretty much our standard